Spiegel Online.js

```diff
@@ -297,21 +278,20 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"notes": [],
-				"tags": [],
-				"seeAlso": [],
+				"date": "07.04.2012",
```

Comment:
I've thought about this a bit. I think we mostly just need to worry about numeric dates (i.e. dates that have months as words should be parsable by Zotero without assistance). In that case, maybe something like `ZU.parseDate(str, '{D}\\.{M}\\.{Y}')` where the format is a pseudo-regexp and `{D}`, `{M}`, `{Y}` would be placeholders for different parts? Should be convenient and flexible enough, I think.